xml/operations-maintenance-upgrade_maintenance.xml

```diff
@@ -0,0 +1,1835 @@
+     following the procedure documented in <xref linkend="DNS-MIGRATE"/>.
+   </para>
+
+   <procedure>
+     <title>Cleanup Orphaned Packages</title>
+     <step>
+       <para>
+         After migrating the deployer node, there are a small number of
+         packages that were installed that are no longer required&mdash;such as
+         the <systemitem class="service">ceilometer</systemitem> and <systemitem class="service">freezer</systemitem> <literal>virtualenv</literal> (venv) packages.
+       </para>
+       <para>
+         You can safely remove these packages with the following command:
+       </para>
+       <screen>&prompt.ardana;zypper packages --orphaned
+Loading repository data...
+Reading installed packages...
+S | Repository | Name                             | Version                           | Arch
+--+------------+----------------------------------+-----------------------------------+-------
+i | @System    | python-flup                      | 1.0.3.dev_20110405-2.10.52        | noarch
+i | @System    | python-happybase                 | 0.9-1.64                          | noarch
+i | @System    | venv-openstack-ceilometer-x86_64 | 9.0.8~dev7-12.24.2                | noarch
+i | @System    | venv-openstack-freezer-x86_64    | 5.0.0.0~xrc2~dev2-10.22.1         | noarch
+ardana> sudo zypper remove venv-openstack-ceilometer-x86_64 venv-openstack-freezer-x86_64
+Loading repository data...
+Reading installed packages...
+Resolving package dependencies...
+
+The following 2 packages are going to be REMOVED:
+  venv-openstack-ceilometer-x86_64 venv-openstack-freezer-x86_64
+
+2 packages to remove.
+After the operation, 79.0 MiB will be freed.
+Continue? [y/n/...? shows all options] (y): y
+(1/2) Removing venv-openstack-ceilometer-x86_64-9.0.8~dev7-12.24.2.noarch ..................................................................[done]
+Additional rpm output:
+/usr/lib/python2.7/site-packages/ardana_packager/indexer.py:148: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
+  return yaml.load(f)
+
+
+(2/2) Removing venv-openstack-freezer-x86_64-5.0.0.0~xrc2~dev2-10.22.1.noarch ..............................................................[done]
+Additional rpm output:
+/usr/lib/python2.7/site-packages/ardana_packager/indexer.py:148: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
+  return yaml.load(f)
+       </screen>
+     </step>
+   </procedure>
+
+   <procedure>
+     <title>Delete freezer Containers from the &o_objstore;</title>
+     <para>
+       The freezer service has been deprecated and removed from &productname; &productnumber; &clm;,
+       but the backups that the &freezer; service created before you upgraded
+       will still be consuming space in your Swift Object store.
```

Comment:
Entity for "Swift object store", or leave it? I think it looks better with a capital... or we could just redefine the entities for Swift and Freezer to start with capitals... What do you think, @asettle & @rtamalin ?

Reply:
@lproven swift is purposefully lowercase :) project name for OpenStack are lowercase, whereas the official service name Object Storage, is uppercase.

Reply:
OK, fair enough then.

Reply:
I'm happy either way; probably better to be consistent and use the entities everywhere?